Make.config

```diff
@@ -44,10 +44,10 @@ include $(TOP)/Make.versions
 # The value is taken from the name + version of the Ref pack.
 # Example: given the Ref pack "Microsoft.iOS.Ref.net8.0_17.5" with the version "17.5.8030", the value
 # to write here would be the part after "Microsoft.iOS.Ref." + "/" + version: "net8.0_17.5/17.5.8030"
-STABLE_NUGET_VERSION_iOS=net8.0_18.0/18.0.8303
-STABLE_NUGET_VERSION_tvOS=net8.0_18.0/18.0.8303
-STABLE_NUGET_VERSION_MacCatalyst=net8.0_18.0/18.0.8303
-STABLE_NUGET_VERSION_macOS=net8.0_15.0/15.0.8303
+STABLE_NUGET_VERSION_iOS=net9.0_18.0/18.0.9615
+STABLE_NUGET_VERSION_tvOS=net9.0_18.0/18.0.9615
+STABLE_NUGET_VERSION_MacCatalyst=net9.0_18.0/18.0.9615
+STABLE_NUGET_VERSION_macOS=net9.0_15.0/15.0.9615
```

Comment:
This won't work until the packages are actually on nuget.org (we download them directly from nuget.org, we don't restore them using package dependencies, which means the feeds in NuGet.config won't be looked at)